sql/core/src/main/scala/org/apache/spark/sql/sources/filters.scala

```diff
@@ -38,6 +38,14 @@ case class EqualTo(attribute: String, value: Any) extends Filter
 
 /**
  * A filter that evaluates to `true` iff the attribute evaluates to a value
+ * null safe and equal to `value`.
```

Comment:
it'd be better if we expand this a bit to explain what null-safe means. e.g. similar to what's in mysql docs: http://dev.mysql.com/doc/refman/5.0/en/comparison-operators.html#operator%5Fequal-to